nsqlookupd/http.go

```diff
@@ -77,13 +77,40 @@ func (s *httpServer) doInfo(w http.ResponseWriter, req *http.Request, ps httprou
 	}, nil
 }
 
+type topicMeta struct {
```

Comment:
I'd prefer these response types be inlined in some way to more clearly indicate that they're not used elsewhere

Reply:
done.